mm2src/coins/utxo/bchd_grpc.rs

```diff
@@ -30,23 +28,17 @@ where
     Res: prost::Message + Default + Send + 'static,
     Url: AsRef<str>,
 {
-    let futures = urls
-        .iter()
-        .map(|url| post_grpc_web::<_, Res>(url.as_ref(), req).map(move |res| (url, res)));
-
-    join_all(futures)
-        .await
-        .into_iter()
-        .map(|(url, response)| {
-            Ok((
-                url,
-                response.mm_err(|err| GrpcWebMultiUrlReqErr {
-                    to_url: url.as_ref().to_string(),
-                    err,
-                })?,
-            ))
-        })
-        .collect()
+    let mut results = Vec::new();
```

Comment:
For some reason, I ran into invalid lifetime parameters when I changed a code not related to the validation of SLP UTXOs. Actually, when I fixed another compile error, this function started working correctly without these changes, but I refactored it already, so I decided to leave it as it is.
I can roll back the changes since it's more about taste

Reply:
It's not just "taste". `join_all` runs the futures in parallel so the approximate execution time will be O(response time of the slowest server). With the `for` loop the futures will be running sequentially so execution time becomes O(N), where N is the number of futures. The difference might be quite significant because it's recommended to use as many sources for SLP validation as possible.

Reply:
Got it! Then I'll return the previous implementation, thank you!

Reply:
Done